src/components/AccountDetails.tsx

```diff
@@ -0,0 +1,111 @@
+import "./AccountDetails.scss"
+
+import React, { ReactElement } from "react"
+import { commify, formatBNToString } from "../utils"
+import Copy from "./Copy"
+import Identicon from "./Identicon"
+import { SUPPORTED_WALLETS } from "../constants"
+import { Zero } from "@ethersproject/constants"
+import { getEtherscanLink } from "../utils/getEtherscanLink"
+import { shortenAddress } from "../utils/shortenAddress"
+import { useActiveWeb3React } from "../hooks"
+import { usePoolTokenBalances } from "../state/wallet/hooks"
+import { useTranslation } from "react-i18next"
+
+interface Props {
+  openOptions: () => void
+}
+
+export default function AccountDetail({ openOptions }: Props): ReactElement {
+  const { t } = useTranslation()
+  const { account, connector } = useActiveWeb3React()
+  const tokenBalances = usePoolTokenBalances()
+  const ethBalanceFormatted = commify(
+    formatBNToString(tokenBalances?.ETH || Zero, 18, 6),
+  )
+
+  const connectorName = Object.keys(SUPPORTED_WALLETS)
+    .filter((k) => SUPPORTED_WALLETS[k].connector === connector)
+    .map((k) => SUPPORTED_WALLETS[k].name)[0]
+
+  console.log(connectorName)
+
+  return (
+    <div className="accountDetail">
+      <div className="upperSection">
+        <h3 className="accountTitle">{t("account")}</h3>
+        <div className="accountControl">
+          <span className="label">Connected with {connectorName}</span>
+          <span className="label">Balance</span>
+          <div className="data">
+            <Identicon />
+            <span className="address">
+              {account && shortenAddress(account)}
+            </span>
+            {account && (
+              <a
+                href={getEtherscanLink(account, "address")}
+                target="_blank"
+                rel="noreferrer"
+              >
+                <svg
+                  width="16"
+                  height="16"
+                  viewBox="0 0 16 16"
+                  fill="none"
+                  xmlns="http://www.w3.org/2000/svg"
+                >
+                  <path d="M11.6667 11.6667H4.33333V4.33333H8V3H4.33333C3.59333 3 3 3.6 3 4.33333V11.6667C3 12.4 3.59333 13 4.33333 13H11.6667C12.4 13 13 12.4 13 11.6667V8H11.6667V11.6667ZM9.33333 2V3.33333H11.7267L6.17333 8.88667L7.11333 9.82667L12.6667 4.27333V6.66667H14V2H9.33333Z" />
+                </svg>
```

Comment:
replace inline svg's with the assets in `src/assets/icons/*.svg`

Reply:
Import asset couldn't change fill color. We apply different fill colors to <svg> to support both themes.